brainglobe_atlasapi/atlas_generation/wrapup.py

```diff
@@ -116,6 +136,11 @@ def wrapup_atlas_from_data(
 
     # If no hemisphere file is given, assume the atlas is symmetric:
     symmetric = hemispheres_stack is None
+    if isinstance(annotation_stack, str) or isinstance(annotation_stack, Path):
+        annotation_stack = tifffile.imread(annotation_stack)
```

Comment:
Why is this needed? Hopefully we'd always read the annotation into an array in `retrieve_reference_and_annotation`

Reply:
We need annotation stack loaded as an array for filter_structures_not_present_in_annotation
https://github.com/brainglobe/brainglobe-atlasapi/blob/d9124c690d9e6f2c8caa58a5067b8006e31df3b9/brainglobe_atlasapi/atlas_generation/wrapup.py#L141-L143

Reply:
Ah, yea, I see. Maybe in the future we want to disallow passing a path or string in the first place, but that's outside the scope of this PR.